### ISSUE TITLE
`tests/ttnn/integration_tests/unet/test_ttnn_shallow_unet.py` failing for WH

### ISSUE THREAD

**Issue by vtangTT** [tenstorrent/tt-metal#7556]
This is a WH only test. Failing with this signature: 

```
FAILED tests/ttnn/integration_tests/unet/test_ttnn_shallow_unet.py::test_unet[perf_mode=False-groups=1-loop=0] - AssertionError: act_block_h must evenly divide out_block_h
FAILED tests/ttnn/integration_tests/unet/test_ttnn_shallow_unet.py::test_unet[perf_mode=True-groups=1-loop=0] - AssertionError: act_block_h must evenly divide out_block_h
```

Going to skip on CI for now

CC: @tt-rkim @TT-billteng

**Comment by tt-rkim** [comment]
@mtatsumiTT is this a test you're aware of?

**Comment by mtatsumiTT** [comment]
hmm no, I'm modifying tests in `tests/ttnn/integration_tests/stable_diffusion/` only

**Comment by tt-rkim** [comment]
@nsmithtt Any insight on this?

**Comment by nsmithtt** [comment]
I'll take a look!

**Comment by TT-billteng** [comment]
if you think this is a regression issue, we've been using https://github.com/tenstorrent/tt-metal/blob/main/tests/scripts/tt_bisect.sh with a lot of success

**Comment by nsmithtt** [comment]
This is because it's running on an x2 board and the test doesn't have an automatic override for `WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml`.  Following up on slack to see what the best approach is here.

**Comment by nsmithtt** [comment]
Blocked by: https://github.com/tenstorrent/tt-metal/issues/7606

**Comment by nsmithtt** [comment]
Unet just doesn't run on x2 devices, marked as skip from this fix 16a01c91565f454ac44b100418fadeef0528f5e1
